client/components/app.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
-import Chart5y from './chart5y.jsx';
+import Chart from './chart.jsx';
```

Comment:
would recommend capitalizing your file names for react components https://github.com/airbnb/javascript/tree/master/react#naming